docs/use-as-modules.md

```diff
@@ -11,29 +11,28 @@
 // Level of abstraction(descending order)
 // cli > TextLintEngine > TextLintCore(textlint)
 // See: https://github.com/textlint/textlint/blob/master/docs/use-as-modules.md
-module.exports = {
-    // Command line interface
-    cli,
-    // TextLintEngine is a wrapper around `textlint` for linting **multiple** files
-    // include formatter, detecting utils
-    // <Recommend>: It is easy to use
-    // You can see engine/textlint-engine-core.js for more detail
-    TextLintEngine,
-    // TextFixEngine is a wrapper around `textlint` for linting **multiple** files
-    // include formatter, detecting utils
-    // <Recommend>: It is easy to use
-    // You can see engine/textlint-engine-core.js for more detail
-    TextFixEngine,
-    // It is a singleton object of TextLintCore
-    // Recommend: use TextLintCore
-    textlint,
-    // Core API for linting a **single** text or file.
-    TextLintCore,
-    // Constant Types
-    TextLintMessageType: MessageType,
-    TextLintMessageSeverityLevel: SeverityLevel,
-    TextLintNodeType
```

Comment:
Some user.
But it is for testing.

- https://github.com/search?utf8=%E2%9C%93&q=TextLintNodeType&type=Code

The user can use `@textlint/ast-node-types` insteadof it.

Reply:
I've filed related issues and note it as BREAKING CHANGE.
https://github.com/textlint/textlint-filter-rule-whitelist/issues/1
https://github.com/textlint/textlint-rule-report-node-types/issues/2
https://github.com/textlint/textlint-filter-rule-node-types/issues/9


📝 textlint 10.0.0 notes

- https://github.com/textlint/textlint/issues/346